src/libratbag-data.c

```diff
@@ -83,8 +89,10 @@ struct ratbag_device_data {
 	int refcount;
 	char *name;
 	char *driver;
+	char *device_type;
```

Comment:
duplicate and risky. the enum carries all the information we want, there's no need to have the extra `char *`

Reply:
ok I just copied the style of how driver is implemented